src/test/java/io/airbrake/javabrake/QueriesTest.java

```diff
@@ -138,33 +131,20 @@ public void testQueryNotifySuccess() {
 
     long ms = this.endTime.getTime() - this.startTime.getTime();
     queryStats.add(ms);
-    Queries queries = new Queries(config.environment, list);
 
-    OkSender okSender = new OkSender(config);
+    Queries queries = new Queries(config.environment, list);
 
+    stubFor(post(urlEqualTo("/api/v5/projects/1/queries-stats")).withHeader("Authorization", containing("Bearer "))
+        .withRequestBody(equalToJson(OkSender.gson.toJson(queries), true, true))
```

Comment:
Nice addition of the `withRequestBody` assertion here 🎉 . Currently it uses the return of `OkSender.gson.toJson(queries)`, the issue is if the queries object structure changed this would still pass, we want changes to the object that cause the posted payload to change to break the test.

Can we assert the `withRequestBody` is called with json as a string that we expect to send instead?

The wiremock docs on [Request Matching > JSON Equality](https://wiremock.org/docs/request-matching/#json-equality) look to provide what we need.
I think we can use a combination of [less strict matching](https://wiremock.org/docs/request-matching/#less-strict-matching) and if there are tricky parts of the JSON we can have a more general matcher with the use of [JsonUnit placeholders](https://wiremock.org/docs/request-matching/#placeholders) e.g.
```java
.withRequestBody(equalToJson({ "id": "${json-unit.any-string}" }, true, true))
// would match { "id": "Sumit'sCoolPR555" }
```

More detail on [JsonUnit placeholder in their docs](https://github.com/lukas-krecan/JsonUnit#type-placeholders)